### ISSUE TITLE
UNABLE TO USE ai RoboForm (R) Version 7.1.3 (en-english WITH nvda

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1334]
**Reported by dharvey on 2011-01-25 09:21**
Hi,

I'm using AI RoboForm (R) Version 7.1.3 (en-english with NVDA to fill in forms on websites. But when I go to make changes to my passcards and/or profile I'm unable to tab around. Problem doesn't occur in AI RoboForm (R) Version 6.10.1 (en-english.

**Comment by ehollig** [comment]
The latest version of RoboForm is 8.3.7. Do we know if the problem still persists? Also, this doesn't sound to be an NVDA bug. It sounds like a RoboForm tabindex problem. More information would be needed though to determine that.
